client/stylesheets/_toCat.scss

```diff
@@ -3302,6 +3310,18 @@ On click the images will be displayed at normal size to complete the effect
     float: right;
 } 
 
+.listingItemLocationString {
+    margin-top: 18px;
+    padding: 10px 0px 10px 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `10px 0px` instead of `10px 0px 10px 0px`<br>`0px` should be written without units as `0`